### ISSUE TITLE
Build warnings after janusgraph connector changes

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#1771]
From build: 
```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.odpi.egeria:developer-resources:jar:1.1-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.apache.tinkerpop:tinkergraph-gremlin:jar -> duplicate declaration of version ${tinkergraph.version} @ org.odpi.egeria:egeria:1.1-SNAPSHOT, /Users/jonesn/IdeaProjects/egeria/pom.xml, line 469, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.apache.lucene:lucene-analyzers-common:jar -> duplicate declaration of version ${lucene-analyzers-common.version} @ org.odpi.egeria:egeria:1.1-SNAPSHOT, /Users/jonesn/IdeaProjects/egeria/pom.xml, line 476, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.janusgraph:janusgraph-core:jar -> duplicate declaration of version ${janus.version} @ org.odpi.egeria:egeria:1.1-SNAPSHOT, /Users/jonesn/IdeaProjects/egeria/pom.xml, line 490, column 25
[WARNING]
[WARNING] Some problems were encountered while building the effective model for org.odpi.egeria:egeria:pom:1.1-SNAPSHOT
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.apache.tinkerpop:tinkergraph-gremlin:jar -> duplicate declaration of version ${tinkergraph.version} @ line 469, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.apache.lucene:lucene-analyzers-common:jar -> duplicate declaration of version ${lucene-analyzers-common.version} @ line 476, column 25
[WARNING] 'dependencyManagement.dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.janusgraph:janusgraph-core:jar -> duplicate declaration of version ${janus.version} @ line 490, column 25
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```

**Comment by planetf1** [comment]
This fix is also needed in 1.1. It appears at the head of every build so is very obvious and looks unprofessional. 

Will also merge into 1.1 branch

**Comment by planetf1** [comment]
closing as fixed on all releases needed
